README.md

```diff
@@ -113,6 +113,43 @@ export default PixiComponent('Rectangle', {
 })
 ```
 
+With ES6:
+
+`./components/Rectangle.mjs`
+
+```
+import { Graphics } from 'pixi.js'
+import { registerComponent } from '@inlet/react-pixi'
+
+export class Rectangle {
+  constructor() {
+    
+  }
+  
+  create = (props) => {
+    return new Graphics()
+  }
+  
+  didMount = (instance, parent) => {
+    // apply custom logic on mount
+  }
+  
+  willUnmount = (instance, parent) => {
+    // clean up before removal
+  }
+    
+  applyProps = (instance, oldProps, newProps) => {
+    const { fill, x, y, width, height } = newProps
+    instance.clear()
+    instance.beginFill(fill)
+    instance.drawRect(x, y, width, height)
+    instance.endFill()
+  }
+}
+
+registerComponent('Rectangle', this)
```

Comment:
This example doesn't work.  I think you'll need call the `registerComponent` in the constructor right?

The docs already states the usage of `PixiComponent`, which is basically the same. I prefer to keep it clean and leave this out of the README.